tests/test_spd_matrices.py

```diff
@@ -73,6 +87,33 @@ def test_random_uniform_and_belongs_vectorization(self):
         result = self.space.belongs(points)
         expected = gs.array([True] * 4)
         self.assertAllClose(result, expected)
+#
+    @geomstats.tests.np_only
+    def test_random_gaussian_rotation_orbit(self):
+        """Test random_gaussian_rotation_orbit"""
+        self.setUp_alt(n=2)
+        mean_spd = self.space.random_uniform()
+        var_rotations = 1.
+        points = self.space.random_gaussian_rotation_orbit(
+            mean_spd=mean_spd, var_rotations=var_rotations, n_samples=4)
+        result = self.space.belongs(points)
+        expected = gs.array([True] * 4)
+        self.assertAllClose(result, expected)
+
+    @geomstats.tests.np_only
```

Comment:
Why only in np, what fails in tf and torch?

Reply:
In both cases an error is thrown while attempting a `vectorize` operation:
`return gs.vectorize([point, alpha], self._multiply_angle_of_rot2,
                            multiple_args=True, signature='(n,n),()->(n,n)')`

tf throws an error, which I've unsuccesfully tried to debug:
> (axis, -value_shape.ndims, value_shape.ndims))
> ValueError: axis = 0 not in [0, 0)

torch backend doesn't seem to have such an operation?
`vectorize = _raise_not_implemented_error`

A solution could be rolling back to vectorize the code through list comprehensions, but perhaps I'm just using the vectorization operator wrong.